Meta/Lagom/CMakeLists.txt

```diff
@@ -102,6 +102,7 @@ endif()
 if (CMAKE_CXX_COMPILER_ID MATCHES "Clang$")
     # Clang's default constexpr-steps limit is 1048576(2^20), GCC doesn't have one
     add_compile_options(-Wno-overloaded-virtual -Wno-user-defined-literals -fconstexpr-steps=16777216)
+    add_compile_options(-Wno-unused-private-field)
```

Comment:
Please add a FIXME, ideally linking the issue mentioned in the commit message

Reply:
The issue itself is not yet created, but I created issue #14137 to document the fact that we need to document these options (and why). I added a comment pointing to that issue for now!